MAUI/Radial-Menu/SfRadialMenuItem-Customization.md

```diff
@@ -729,3 +729,53 @@ namespace RadialMenuCommandSample
 }
 
 {% endhighlight %}
+
+## Font autoscaling enabled
+
+The `FontAutoScalingEnabled` property is used to automatically scale the control's font size based on the operating system's text size. The default value of `FontAutoSaclingEnabled` property is `false`.
```

Comment:
The `FontAutoScalingEnabled` property is used to automatically scale the Radial menu item's font size based on the operating system's text size. The default value of the `FontAutoSaclingEnabled` property is `false.`

Reply:
Yes, changed.